FrameworkSystem/DB/ProxyDB.py

```diff
@@ -1064,9 +1064,12 @@ def setPersistencyFlag(self, userDN, userGroup, persistent=True):
       if not result['OK']:
         self.log.error("setPersistencyFlag: Can not retrieve username for DN", userDN)
         return result
-      userName = result['Value']
+      try:
+        sUserName = self._escapeString(result['Value'])['Value']
+      except KeyError:
```

Comment:
Our usual construct in this case is:

   result = self._escapeString(strValue)
   if not result['OK']:
     return result
   userName = result['Value']

Reply:
And why userName -> sUserName ? according to coding conventions we use Camel style and not the hungarian one

Reply:
I copied the style of the existing code
```
   try:
      sUserDN = self._escapeString(userDN)['Value']
      sUserGroup = self._escapeString(userGroup)['Value']
    except KeyError:
      return S_ERROR("Can't escape something")
```
`s` in `sUserName` means that to user name added `""` quotes

Reply:
Ah, I see, this is Adri's style who loved hungarian notation. OK, leave it like that